.github/workflows/try.yml

```diff
@@ -20,15 +28,34 @@ jobs:
         with:
           clean: False
 
-      - name: Clone qibo repository
-        uses: actions/checkout@v2
+      #- name: Clone qibo repository
+      #  uses: actions/checkout@v2
+      #  with:
+      #    repository: qibogang/qibo
```

Comment:
Since the repo running will be `qibo` (or whatever other one with the docs) you can also kill this one forever :)